tests/test_health.py

```
@@ -229,18 +253,126 @@ async def test_health_setup_action_returns_declined_message() -> None:
-    from unittest.mock import AsyncMock, MagicMock, patch
-
     from unraid_mcp.config.settings import CREDENTIALS_ENV_PATH
 
     tool_fn = _make_tool()
 
-    with patch("unraid_mcp.tools.health.elicit_and_configure", new=AsyncMock(return_value=False)):
-        result = await tool_fn(action="setup", ctx=MagicMock())
+    real_path_str = str(CREDENTIALS_ENV_PATH)
+    mock_path = MagicMock()
+    mock_path.exists.return_value = False
+    type(mock_path).__str__ = lambda self: real_path_str  # type: ignore[method-assign]
```

Ruff (0.15.5)

[warning] Unused lambda argument: `self`

(ARG005)